aspnetcore/security/authentication/community.md

```diff
@@ -21,8 +21,9 @@ The list below is sorted alphabetically.
 | Name | Description |
 | ---- | ----------- |
 | [AspNet.Security.OpenIdConnect.Server (ASOS)](https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server) | ASOS is a low-level, protocol-first OpenID Connect server framework for ASP.NET Core and OWIN/Katana. |
+| [Cierge](https://github.com/pwdless/Cierge) | Cierge is an OpenID Connect server that handles user signup, login, profiles, management, and social logins. |
+| [Gluu Server](https://gluu.org/) | Enterprise ready, open source software for identity, access management (IAM), and single sign-on (SSO). For more information, see the [Gluu Product Documentation](https://gluu.org/docs/). |
 | [IdentityServer](https://identityserver.io/) | IdentityServer is an OpenID Connect and OAuth 2.0 framework for ASP.NET Core, officially certified by the OpenID Foundation and under governance of the .NET Foundation. For more information, see [Welcome to IdentityServer4 (Documentation)](https://identityserver4.readthedocs.io/en/release/). |
-| [OpenIddict](https://github.com/openiddict/openiddict-core) | OpenIddict is an easy-to-use OpenID Connect server for ASP.NET Core. |
-| [Cierge](https://github.com/pwdless/Cierge) | Cierge is an OpenID Connect server that handles user signup, login, profiles, management, social logins, and more. |
+| [OpenIddict](https://github.com/openiddict/openiddict-core) | OpenIddict is an OpenID Connect server for ASP.NET Core. |
```

Comment:
I'm curious to know why you've changed the description of a project you don't own :open_mouth:

Reply:
The Microsoft style guidelines don't permit the docs to make the claim that use of OpenIddict is "easy" (although I *do agree with you* that OpenIddict is **easy to use** :+1:). Here's the guideline ...

> Take care to avoid words and phrases that are subjective, like easy, quick, and seamless, or overly superlative, like world-class, lightning-fast, awesome, and amazing.

This rule is strictly enforced by @Rick-Anderson. He's stopped many authors and contributors from using words like "easy" and "simple" in various ways, and I imagined that this just fell through the cracks. I even seem to recall him saying that these types of words are "banned" from this repo in one convo about it. If he'll permit it in this case, it's totally cool with me. He's the boss!!

RE: description of a project you don't own

Well, I think you're saying it's "bad form" and perhaps "disrespectful" to change this description, especially without discussing it with you first. I'm sorry about that. I really didn't think it was that big of a deal. I wouldn't permit any author/contributor to break an established style manual guideline without management's permission. I've stricken other such words ... For example just the other day, a contributor submitted the word "free" in a product/service description, and I had to remove that, too. We can't go saying "this is free;" and then later, it isn't free and someone gets very cross with us for telling them the thing was "free." We have another guideline to follow in that case.

No disrespect was intended. I'm just trying to follow the guidelines as best I can. Your description ("easy-to-use") is a violation of a guideline that all docs must follow ... all of the text ... all of the descriptions ... everything ... no matter the source or purpose.

Reply:
> Your description ("easy-to-use") is a violation of a guideline that all docs must follow ... all of the text ... all of the descriptions ... everything ... no matter the source or purpose.

Actually, it's not "my" description as this entry was added by [@]sayedihashimi [as part of this PR](https://github.com/aspnet/Docs/pull/3314), that was reviewed and approved by [@]danroth27 (and me). But maybe the guidelines have changed since then?

I'm not blaming you for respecting the guidelines (tho' I'm not sure why they should apply to product descriptions: this doc is not exactly what I'd call "guidance"). It's just surprising you've decided to change that a year after without reaching me first, here or on Slack.

Reply:
I see. Of course, we honor whatever `@danroth27` approves. In that case, @Rick-Anderson has been informed. I'll change it back.

For the community-at-large, I don't want to set a precedence here. We'll follow the guidelines for all content submissions, including 3rd party product/service descriptions. We'll strike those words and phrases that the guidelines don't permit unless we have management's approval. The guidelines don't distinguish between "programming guidance" and "product/service descriptions." If we have management's approval to circumvent the guidelines, that's 🎷 with us. :smile: